cognee/api/client.py

```
@@ -65,9 +52,12 @@ async def lifespan(app: FastAPI):
 
 from cognee.api.v1.users.routers import get_auth_router, get_register_router,\
     get_reset_password_router, get_verify_router, get_users_router
-
-from cognee.api.v1.permissions.get_permissions_router import get_permissions_router
-
+from cognee.api.v1.permissions.routers import get_permissions_router
```

Ruff

Module level import not at top of file

(E402)